.gitignore

```diff
@@ -14,6 +17,7 @@
 # Untracked files
 config.yaml
 dag.svg
+scripts/__init__.py
```

Comment:
Good catch. This was a test as I didn't want a license note in the __init__.py.
Having now second thoughts, it might make sense to have a license note there anyways.
The `__init__.py` is not always empty. 

Therefore, yes, the `__init__.py` should be removed here.